.github/workflows/build.yml

```diff
@@ -148,6 +148,11 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ inputs.configure-aws-region }}
+      - name: Login to Docker
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
you have to list the secrets at the "secrets: " section

Reply:
Add it to the section. Thanks